### PR TITLE
xrootd 4.12.5

### PR DESCRIPTION
**Pull request by mprince**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
https://github.com/xrootd/xrootd/blob/master/LICENSE

```
"XRootD is free software: you can redistribute it and/or modify it under the terms \n"
"of the GNU Lesser General Public License as published by the Free Software        \n"
"Foundation, either version 3 of the License, or (at your option) any later version.\n\n"
"XRootD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;\n"
"without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR  \n"
"PURPOSE.  See the GNU Lesser General Public License for more details.            \nn"
"You should have received a copy of the GNU Lesser General Public License along    \n"
"with XRootD in a file called COPYING.LESSER (LGPL license) and file COPYING (GPL  \n"
"license).  If not, see <http://www.gnu.org/licenses/>.\n\n"
"Prior to September 2nd, 2012 the XRootD software suite was licensed under a\n"
"modified BSD license (see file COPYING.BSD).  This applies to all code that\n"
"was in the XRootD git repository prior to that date.\n"
```

File: Formula/xrootd.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Xrootd < Formula
   desc "High performance, scalable, fault-tolerant access to data"
   homepage "https://xrootd.slac.stanford.edu/"
-  url "https://xrootd.slac.stanford.edu/download/v4.12.3/xrootd-4.12.3.tar.gz"
-  sha256 "6f2ca1accc8d49d605706bb556777c753860bf46d845b1ee11393a5cb5987f15"
+  url "https://xrootd.slac.stanford.edu/download/v4.12.5/xrootd-4.12.5.tar.gz"
+  sha256 "164c83171e0bfc4d15ed55abde5f8b4d6413aa516c6b1faefabca0e6ba18275f"
+  license "LGPL-3.0"
```
Comment (fxcoudert) [inline Formula/xrootd.rb:6]:
```suggestion
  license "LGPL-3.0-or-later"
```

Comment (mprince) [reply Formula/xrootd.rb]: @fxcoudert should I squash the commits?

Comment (chenrui333) [reply Formula/xrootd.rb]: No need, the CI would auto squash commits now.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @mprince!

**Pull request closed without merging**